completers/nix-build_completer/cmd/root.go

```diff
@@ -19,8 +19,8 @@ func Execute() error {
 func init() {
 	carapace.Gen(rootCmd).Standalone()
 
-	rootCmd.Flags().StringSlice("arg", []string{}, "Pass [name] [expr] to Nix functions.")
-	rootCmd.Flags().StringSlice("argstr", []string{}, "Pass [name] [string] to Nix functions")
+	arg := rootCmd.Flags().StringSlice("arg", []string{}, "Pass [name] [expr] to Nix functions.")
+	argstr := rootCmd.Flags().StringSlice("argstr", []string{}, "Pass [name] [string] to Nix functions")
```

Comment:
I need to pass these in to the attribute completer in case the file we're evaluating takes arguments. I think this is the right way to get the value out?

Reply:
Yes, but i would prefer accessing them like this at the moment:
```go
rootCmd.Flags().GetStringSlice("arg")
```

I have crude script checking/formatting the  order of flags and i am not sure if if the regex matches the variant above (can be updated of course, but that's how it is currently).

Reply:
I think it does work even with assignment as I did see errors referencing them from `caralint` but I've updated them to not be assignments for consistency.